src/main/java/src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,8 @@
+package src.main.java.duke;
+
+public class DukeException extends Exception {
```

Comment:
Perhaps you can include descriptive header comments for the class?
I noticed the same issue in multiple places